app/models/folio/item.rb

```diff
@@ -132,12 +135,47 @@ def paged?
     end
 
     def hold_recallable?
-      HOLD_RECALL_STATUSES.include?(status)
+      recallable? || holdable?
     end
 
-    # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
+    def recallable?
+      HOLD_RECALL_STATUSES.include?(status) && allowed_request_types.include?('Recall')
+    end
+
+    def holdable?
+      HOLD_RECALL_STATUSES.include?(status) && allowed_request_types.include?('Hold')
```

Comment:
allowed_request_types could be nil, so you can't send `include?` to it without checking for nil.